github-pr

```diff
@@ -12,11 +12,19 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    commit_statuses = []
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            commit_statuses.append(commit_status.state)
+    return commit_statuses[-1]
+
 def _print_pr(pr, **args):
     if args['numberonly']:
         print "%d" % pr.number
     else:
-        print "#%d [%s] %10s <- %-30s    %s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title)
+        commit_status = _return_status(pr.get_commits())
+        print "#%d [%s] %10s <- %-30s    %s - Commit/Build Status:%s" % (pr.number, pr.state, pr.base.ref, pr.head.ref, pr.title, commit_status)
```

Comment:
using tabulate is a much larger ticket, since we dont currently return a list but instead call the _print_pr method. Another ticket will be needed to do this. for now ill just remove any print statements and just print the status. I was afraid that it would be hard to show the diff between PR status vs commit status but PR status is either open or closed, not success/failure. this shouldnt be as confusing as I thought